app/payload_mapper.py

```
@@ -0,0 +1,84 @@
+
+
+def _safe_int(value: Any, default: int = 0) -> int:
+    try:
+        if value is None:
+            return default
+        return int(value)
+    except (TypeError, ValueError):
+        return default
+
+
+def _safe_bool(value: Any, default: bool = False) -> bool:
```

Ruff (0.15.15)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)